README.md

```diff
@@ -2,6 +2,20 @@
 
 This is a list of sql commands to add,insert and filter fictional animal elements inside a DB by the name 'vet_clinic'.
 
+# INDEXING
+
+**BEFORE**
+
+![animals_id](./Before/before_performance_animalsid.png)
+![vets_id](./Before/before_performance_vetsid.png)
+![ownersmail](./Before/before_performance_ownermail.png)
+
+**AFTER**
+
+![animals_id](./After/after_performance_animalsid.png)
+![vets_id](./After/after_performance_vetsid.png)
+![ownersmail](./After/after_performance_ownersmail.png)
```

Comment:
- [ ] I would suggest you pair these screenshots into pairs of Before and After so that it will be easier to interpret them, currently they are ordered in a hard to read way, the three before and the three after, it will be more easier if the before is paired up with its after version, kindly arrange it like that, for making context more louder. Thank you!